frontend/static/css/style.css

```diff
@@ -93,16 +65,13 @@ a.hash-link:hover {
   text-decoration: underline;
 }
 
-h2 {
-  margin-top: 0;
-}
-
 .hidden {
   display: none;
 }
 
 .header-example {
-  background: rgba(0, 0, 0, 0.1);
+  background-color: rgba(0, 0, 0, 0.1);
+  color: #757575;
```

Comment:
This reduces the contrast, right? Could we keep the current color?

Reply:
The previous color doesn't have a contrast ration > 4.5 to be WCAG2AA valid.

The difference is subtle, IMO it should be kept like I have it.

Reply:
...but anyway, it's ultimately your call so I'll revert it.

Reply:
The current color is `#424242`. That's darker ⇒ more contrast, right?

Reply:
Yup. Must be a bug in the tool I used (HTM_Codesniffer) to check for this.